server/services/homekit/index.js

```diff
@@ -0,0 +1,43 @@
+const logger = require('../../utils/logger');
+const HomeKitHandler = require('./lib');
+const HomeKitController = require('./api/homekit.controller');
+
+module.exports = function HomeKitService(gladys, serviceId) {
+  const hap = require('hap-nodejs');
+
+  const homeKitHandler = new HomeKitHandler(gladys, serviceId, hap);
+
+  /**
+   * @public
+   * @description This function starts the HomeKit service and expose devices
+   * @returns {Promise} Empty promise once service is started.
+   * @example
+   * gladys.services.homekit.start();
+   */
+  async function start() {
+    logger.info('Starting HomeKit service');
+    const dockerBased = await gladys.system.isDocker();
+    if (dockerBased) {
+      const { basePathOnContainer } = await gladys.system.getGladysBasePath();
+      hap.HAPStorage.setCustomStoragePath(`${basePathOnContainer}/homekit`);
```

Comment:
Could you cover this with tests? You can mock gladys.system.isDocker to return true